RadixWallet/Features/TransactionReviewFeature/SubmitTransaction/SubmitTransaction.swift

```diff
@@ -151,6 +155,9 @@ public struct SubmitTransaction: Sendable, FeatureReducer {
 
 		case let .statusUpdate(status):
 			state.status = status
+			if state.isAccountLockerClaim, status.isFinal {
+				accountLockersClient.forceRefresh()
+			}
```

Comment:
This could, and should be part of accountLockersClient itself, take as reference the [ThirdPartyDeposits manifest submission](https://github.com/radixdlt/babylon-wallet-ios/blob/95344767b4d24a8b8ff678e4e1fb01c4c253f1ce/RadixWallet/Features/AccountPreferencesFeature/Children/ThirdPartyDeposits/ThirdPartyDeposits%2BReducer.swift#L133)

Reply:
oh nice! I wanted to do it this way, but I didn't know about the existence of `submitTXClient.hasTXBeenCommittedSuccessfully()`, which allows me to await for it inside the `AccountLockersClient`. 

Updated solution to force refresh after successful and failing transactions